gateway/src/shard/processor/impl.rs

```diff
@@ -1046,7 +1023,6 @@ impl ShardProcessor {
         self.resume = Some((seq, id));
 
         if let Err(_source) = self.try_resume().await {
```

Comment:
And in `tracing::warn!` call
```suggestion
        if let Err(source) = self.try_resume().await {